Kenan-Modpack/More_Military/hecu.json

```diff
@@ -287,11 +287,5 @@
     "type": "ARMOR",
     "name": { "str": "H.E.C.U backpack" },
     "description": "Based on the MOLLE backpack's design, this backpack strikes a fine balance between storage space and encumbrance, while allowing for better mobility."
-  },
-  {
-    "type": "scenario",
-    "id": "lab_chal",
-    "copy-from": "lab_chal",
-    "extend": { "professions": [ "hecu" ] }
```

Comment:
why this caused error? I thought we need to remove a JSON with id: hecu

Reply:
oh, I know, starting scenerio will add professions to the list, and this `extend` add "hecu" again, so no `extend` in `     "type": "scenario", `

Reply:
no `extend` in `"type": "scenario",` is allowded